client/my-sites/sidebar/sidebar.jsx

```diff
@@ -172,7 +175,7 @@ export const MySitesSidebar = React.createClass( {
 			jetpackEnabled = config.isEnabled( 'manage/themes-jetpack' ),
 			themesLink;
 
-		if ( site && ! site.isCustomizable() ) {
+		if ( site && ! ( site.capabilities && site.capabilities.edit_theme_options ) ) {
```

Comment:
Little update for anybody who stumbles across this:
`canCurrentUser` now lives in `state/selectors` :)